environment.yml

```
@@ -25,7 +25,7 @@ dependencies:
   - nbval
   - scikit-learn
   - pykdtree
-  - zarr>=2.11.0,<2.18.0
+  - zarr>=2.11.0
```

If you like, it may be worth excluding the problematic version as a safety measure

```suggestion
  - zarr>=2.11.0,!=2.18.0
```